### PR TITLE
Update experiments-msmarco-passage.md with building commands

### PR DESCRIPTION
**Pull request by sahel-sh**
This cl:

- Adds a build command for building Anserini
- Adds commands for extraction and building of trec_tool
- Adds Repro log for sahel-sh

### CODE REVIEW

**Review by lintool — COMMENTED** [review]
What if we just link to the build instructions here?
https://github.com/castorini/anserini#-getting-started

**Comment by sahel-sh** [comment]
> What if we just link to the build instructions here? https://github.com/castorini/anserini#-getting-started

done

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -236,6 +239,14 @@ python tools/scripts/msmarco/convert_msmarco_to_trec_qrels.py \
   --output collections/msmarco-passage/qrels.dev.small.trec
 ```
 
+If you are using the `trec_eval` tool for the first time, you need to extract and build it first:
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:242]: Do we want this instruction explicitly here? it's part of the getting started right?

Comment (sahel-sh) [reply docs/experiments-msmarco-passage.md]: > Do we want this instruction explicitly here? it's part of the getting started right?

had missed it, removed.

**Review by lintool — APPROVED** [review]
lgtm!

**Pull request merged**